src/routes/areas/createArea.ts

```diff
@@ -40,9 +40,11 @@ export const handler: RouteHandler<CreateAreaRoute> = async (req, reply) => {
     reply.status(200).send();
   } catch (error) {
     return reply.status(500).send({
-      error: `${error}`,
-      statusCode: 500,
-      message: "An unknown error occurred when trying to create an Area."
+      error: {
+        error: `${error}`,
+        code: "500",
```

Comment:
the code is not the statusCode, it's something like InvalidId, NotAllowed, etc.
in this case, we want a code like: **unknown**